src/go/plugin/go.d/modules/ping/metadata.yaml

```diff
@@ -22,25 +22,30 @@ modules:
       data_collection:
         metrics_description: |
           This module measures round-trip time and packet loss by sending ping messages to network hosts.
-          
+
           There are two operational modes:
-          
-          - privileged (send raw ICMP ping, default). Requires
-            CAP_NET_RAW [capability](https://man7.org/linux/man-pages/man7/capabilities.7.html) or root privileges:
-            > **Note**: set automatically during Netdata installation.
-          
-            ```bash
-            sudo setcap CAP_NET_RAW=eip <INSTALL_PREFIX>/usr/libexec/netdata/plugins.d/go.d.plugin
-            ```
-          
-          - unprivileged (send UDP ping, Linux only).
-            Requires configuring [ping_group_range](https://www.man7.org/linux/man-pages/man7/icmp.7.html):
-          
+
+          - **Privileged** (send raw ICMP ping, default). Requires the necessary permissions ([CAP_NET_RAW](https://man7.org/linux/man-pages/man7/capabilities.7.html) on Linux, `setuid` bit on other systems).
+
+            These permissions are **automatically** set during Netdata installation. However, if you need to set them manually:
```

Comment:
Hey @ilyam8 

So concerning the `- set `setuid` bit (Other OS)` line.
When I got the automatic update `v1.47.5` in FreeBSD through crontab (periodic), the `netdata-updater.sh`, my `go.d.plugin` lost the `setuid` bit. So every time after the update I need to set it up manually.
Is this something fixed in `v1.47.6`?

Reply:
No, you don’t. If the bit is not set that is a bug.

Reply:
Fixed in #18876